frontend/src/concepts/pipelines/content/ViewPipelineServerModal.tsx

```diff
@@ -9,7 +9,6 @@ import {
   Title,
 } from '@patternfly/react-core';
 import { usePipelinesAPI } from '~/concepts/pipelines/context';
-import EnsureAPIAvailability from '~/concepts/pipelines/EnsureAPIAvailability';
```

Comment:
Just dropped this -- while the server is loading, you may want to look at your creds you used.